website/versioned_docs/version-1.11.0/operator-docs/about-operator-docs.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Is this all backporting this to 1.11?

Reply:
Yes. My comment earlier was incomplete and I just updated it. There's a bug with versioning and `headerLink.js` where adding a new headerLink in a new version causes the building of previous versions to fail.

Reply:
I don't see any support/issues for it on their GitHub. I'll try building locally with a newer version and see if it has been addressed.

Reply:
I should also note that this will port the `about-operator-docs.md` and `cumulus-dependencies.md` files to all previously published documentation versions.